packages/performance/src/web.ts

```diff
@@ -54,4 +65,74 @@ export class FirebasePerformanceWeb
     };
     return result;
   }
+
+  public async putAttribute({
+    traceName,
+    attribute,
+    value,
+  }: PutAttributeOptions): Promise<void> {
+    const trace = this.traces[traceName];
+    if (!trace) return;
```

Comment:
```suggestion
    if (!trace) {
    	return;
    }
```

I don't like the short version. Please always write it like this. ^^ (I need to expand the ESLint rules from Ionic.)